src/stories/standalone/Selector/MultiWithCheckBox.story.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { withActions } from "@storybook/addon-actions";
+import { boolean, withKnobs } from "@storybook/addon-knobs";
+import { MultiSelectWithCheckBox } from "../../../standalone/Selector";
+
+interface MultiSelectOption {
```

Comment:
instead of copy-pasting this from the component class to here, export it and import it here